doc/data-warehouse-what-and-why.md

```diff
@@ -0,0 +1,258 @@
+creating analytical reports for content designers throughout all GOV.UK
+organizations.
+```
+
+So, it seems that from a definition / formal perspective, it fits in what we
+need to track metrics through time.
+
+## Track content changes over time for Content Items
+
+We not only need to track metrics about how content is performing through time,
+but we also need to monitor the content attributes, so we understand the
+relationship between content and performance changes through time.
+
+Say, for example, that a content item was created and published three years ago.
+To get a good understanding of how the content changes are related to their
+performance, we would need to:
+
+1. Track each change to the Content Item through the three years
+2. Update the quality metrics each time a new version is created
+3. Link the performance metrics to each one of the versions of the Content Item.
+
+Tracking the history of each content item is key to understand the relationship
+between content and performance.
+
+## A new application for GOV.UK?
+
+With this context, we decided to build a new application for GOV.UK, a Data
+Warehouse that will track content and performance changes through time and that
+will enable other applications within GOV.UK to get the most of this
+information.
+
+Before building the application, we first checked for the state of the art of
+all the different products within GOV.UK checking if we could re-use some of
+this information, so we don't reinvent the wheel.
+
+This is what we found out:
+
+1. We don't have a single place to retrieve all metrics for content items. We
+   currently store metrics for organizations (and dashboards) via the
+   performance platform, but the way it is structured and the scope of the
+   product, make it not a good fit for our mission goal.
+
+1. We store the number of page views in [Search][rummager], so when users search
+   in GOV.UK the results are ordered by popularity; the way this information is
+   collected [has been reported as Tech debt][ga-tech-debug-trello] and building
+   this applpcation will enable [Search][rummager] to use the API and pay the
+   tech debt.
+
+1. Data Scientists find it difficult to get and link content and performance
+   metrics. Having a single place to look for this information will make their
+   daily work easier.
+
+1. Performance Analysts dashboards will also benefit from having metrics from
+   multiple sources because, although they have direct access to metrics from
+   Google Analytics, they don't have direct access to Feedex or other internal
+   applications.
+
+In summary, we looked at the main applications in GOV.UK ([Content
+Store][content-store],[Publishing API][publishing-api],[Rummager][rummager]),
+and we couldn't find a way to provide the information that we needed to fulfill
+the mission goals. As a side benefit, by satisfying the mission goals, we also
+find out that we were making GOV.UK suite of applications richer.
```

Comment:
Think the last sentence could be clearer / the point it is making isn't very strong.

Reply:
I have deleted the last paragraph. It would need further explanation to make it clear for everyone.

Reply:
Yup - sorry I meant this sentence
```
As a side benefit, by satisfying the mission goals, we also find out that we were making GOV.UK suite of applications richer
```

Reply:
Yeah, I forgot to remove this paragraph. I have just deleted it.